### DISCUSSION TITLE
what's the differences between docker image tag xxx and xxx-dev

### DISCUSSION THREAD

**Discussion by shaoyucheng** [apache/superset#18360] [Q&A / Help] [answered]
I used docker-compose-non-dev.yaml from master to setup my superset environment，and found that the image version was configured as latest-dev.  I browsed the image list in docker hub,  it seems that every tag has a matching -dev tag which presenting a bigger image size. could you please tell me the differences between them? and which one is more recommendable to be used in production env?

**Comment by tberriot** [comment, marked as answer]
Hi @shaoyucheng 
[this](https://github.com/apache/superset/issues/14342) could explain the case

**Reply by shaoyucheng** [reply]
got it and thanks for reply
